documentation/docs/instructor/assessment.mdx

```diff
@@ -0,0 +1,207 @@
+import lockImg from './assets/assessment/lock.png';
+import leaderboardImg from './assets/assessment/leaderboard.png';
+import Image, {ImageSize} from "../../src/components/Image/Image";
+
+# Assessment and Grading
+
+## Overview
+
+Artemis offers three different modes of exercise assessment:
+
+- **Manual:** Reviewers must manually grade student submissions.
+- **Automatic:** Artemis automatically grades student submissions (e.g., via test cases for programming exercises or answer checking for quizzes).
+- **Semi-Automatic:** Artemis provides an automatic starting point that reviewers can manually improve afterward.
+
+<Callout variant={CalloutVariant.info}>
+    <p>Manual assessment is available for programming, text, modeling, and file upload exercises. Quiz exercises use automatic assessment only.</p>
+</Callout>
+
+
+## Manual Assessment Process
+
+Manual assessment involves the following workflow:
+
+1. **Submission:** Students submit their assignments through Artemis.
+2. **Review:** Reviewers access submitted work and place a **lock** on the submission to prevent conflicts with other reviewers.
+3. **Evaluation:** Reviewers assign scores and provide feedback based on established grading criteria.
+4. **Submission:** Reviewers submit their assessment (or cancel to unlock for others).
+5. **Student Feedback:** Students can [rate feedback quality](/student/assessment#rating-feedback-quality), motivating high-quality assessments that helps students understand misconceptions and how to improve.
+
+
+## Assessment Dashboard
+
+The assessment dashboard provides an overview of assessment progress across all manually assessed exercises in a course.
+
+The assessment dashboard helps to track the assessment progress for each exercise.
+
+You can find the assessment dashboard via the **Assessment** button in the course management tab.
+
+<Image src={assessmentDashboardButtonImg} alt={"Assessment Dashboard Button"} size={ImageSize.large}></Image>
+
+This will navigate you to the course-wide Assessment Dashboard.
+
+<Image src={assessmentDashboardImg} alt={"Course-wide Assessment Dashboard"} size={ImageSize.large}></Image>
+
+The course-wide Assessment Dashboard displays:
+- Total number of submissions
+- Number of assessed submissions
+- Number of complaints and feedback requests
+- Average student rating for each exercise
+
+Each exercise has its own **Exercise Dashboard** accessible from this view, showing detailed information for that specific exercise.
+
+<Image src={exerciseDashboardImg} alt={"Exercise Dashboard"} size={ImageSize.large}></Image>
+
+<Callout variant={CalloutVariant.info}>
+    <p>Locks on submissions automatically expire after 24 hours, but reviewers can also unlock manually if needed.</p>
+</Callout>
+
+
+## Structured Grading Criteria
+
+Artemis supports **structured grading instructions** _(comparable to grading rubrics)_ to ensure consistency, fairness and transparency in the grading process.
```

Comment:
```suggestion
Artemis supports **structured grading instructions** (comparable to grading rubrics) to ensure consistency, fairness and transparency in the grading process.
```
Emphasis here looks weird and I do not think it is needed.

Reply:
https://github.com/ls1intum/Artemis/pull/11676/commits/f245fe408c1411db23e4b70a1c78b863bc57afe6